lib/conekta-button.scss

```diff
@@ -1,242 +1,207 @@
-
-$modal_name : "c-modal";
-$font-family: "Montserrat", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+$font-family : "Montserrat", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+$modal_name  : "c-modal";
 
 @import url(http://fonts.googleapis.com/css?family=Montserrat);
 
-.#{$modal_name} * { 
-	box-sizing: border-box;
+.#{$modal_name} * {
+	box-sizing : border-box;
 }
 
-.#{$modal_name}{
-	display: block;
-	font-size: 62.5%; 
-	
-	overflow-x: auto;
-	overflow-y: scroll;
-	position: fixed;
-	top: 0px;
-	right: 0px;
-	bottom: 0px;
-	left: 0px;
-	z-index: 1050;
-
-	padding-right: 25px;
-	padding-left: 25px;
-	outline: 0px none;
-	opacity: 1;
-	background: rgba(0,0,0,0.4);
-	font-family: $font-family;
-
-	.#{$modal_name}-dialog{
-		position: relative;
-		width: 413px;
-		margin: 5% auto;
-		margin-top: 104px;
-		background: #FFFFFF;
-		border-radius: 4px;
-		box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.5);
-
-		.#{$modal_name}-content{
-
-			.#{$modal_name}-header{
-				margin: 0 26px;
-				padding-top: 22px;
-
-				h1{
-					margin: 0;
-					font-size: 27px;
-					letter-spacing: .01em;
-
+.#{$modal_name} {
+	background  : rgba(0, 0, 0, 0.4);
+	bottom      : 0px;
+	display     : block;
+	font-family : $font-family;
+	font-size   : 62.5%;
+	left        : 0px;
+	opacity     : 1;
+	outline     : 0px none;
+	overflow-x  : auto;
+	overflow-y  : scroll;
+	padding     : 0 25px;
+	position    : fixed;
+	right       : 0px;
+	top         : 0px;
+	z-index     : 1050;
+
+	.#{$modal_name}-dialog {
+		background    : #FFFFFF;
+		border-radius : 4px;
+		box-shadow    : 0px 5px 15px rgba(0, 0, 0, 0.5);
+		margin        : 104px auto 5%;
+		position      : relative;
+		width         : 413px;
+
+		.#{$modal_name}-content {
+			.#{$modal_name}-header {
+				margin      : 0 26px;
+				padding-top : 22px;
+
+				h1 {
+					font-size      : 27px;
+					letter-spacing : .01em;
+					margin         : 0;
 				}
 			}
 
-			.#{$modal_name}-body{
-				margin: 0 26px;
+			.#{$modal_name}-body {
+				margin : 0 26px;
 			}
 
-			.#{$modal_name}-footer{
-				padding: 20px 26px 13px;
-				border-bottom-right-radius: 4px;
-				border-bottom-left-radius: 4px;
-				
-				background-color: #F9FAFB;
-				border-top: 1px solid #EEEEEE;
-
-
-				button{
-					font-family: $font-family;
-
-					display: inline-block;
-					transition: all 0.15s ease-out 0s;
-					border-radius: 6px;
-					color: white;
-					font-weight: bold;
-					letter-spacing: 1px;
-					margin-bottom: 1em;
-					padding: 1em;
-					text-align: center;
-					border: none;
-					box-shadow: none;
-					text-transform: uppercase;
-
-					&#paynow{
-						background-color: #6D48CC;
-						&:hover{
-							background-color: darken(#6D48CC, 10%);
+			.#{$modal_name}-footer {
+				background-color : #F9FAFB;
```

Comment:
Color `#F9FAFB` should be written as `#f9fafb`<br>Property name should be immediately followed by a colon